src/app/layout.tsx

```diff
@@ -11,26 +10,6 @@ import { Footer } from "@/shared/components/footer";
 import { Header } from "@/shared/components/header";
 import { ThemeRegistry } from "@/shared/theme/theme-registry";
 import { AuthenticationProvider } from "./_providers/authentication-provider";
-import type { Metadata } from "next";
-
-const description = [
-  "BUILD is a meme and a social game designed to reward builders via",
-  "onchain nominations.",
-].join(" ");
-
-export async function generateMetadata(): Promise<Metadata> {
-  return {
-    title: "BUILD",
-    description: description,
-    openGraph: {
-      title: "BUILD",
-      description: description,
-      type: "website",
-      url: "https://build.top",
-      images: ["https://build.top/images/BUILD-thumbnail.jpg"],
-    },
-  };
-}
```

Comment:
why did this disappear? we will loose metadata on all pages but the 2 we are setting it now. Metadata is incremental. We cna keep here everything that is common, and add, per page, the tidbits that are necessary for frames.